src/script/script.h

```diff
@@ -38,6 +38,12 @@ static const int MAX_STACK_SIZE = 1000;
 // otherwise as UNIX timestamp.
 static const unsigned int LOCKTIME_THRESHOLD = 500000000; // Tue Nov  5 00:53:20 1985 UTC
 
+// Maximum nLockTime. Since a lock time indicates the last invalid timestamp, a
+// transaction with this lock time will never be valid unless lock time
+// checking is disabled (by setting all input sequence numbers to
+// SEQUENCE_FINAL).
+static const uint32_t LOCKTIME_MAX = 0xffffffff;
```

Comment:
0xFFFFFFFFU  ? lest we immediately get another hysterically titled "prevent integer overflow" PR?

Reply:
@gmaxwell Why would an `U` suffix be needed for an integer literal with a hexadecimal base?